wire/client_test.go

```diff
@@ -290,22 +338,42 @@ func TestWireProcess(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	for cname, makeProc := range map[string]jasper.ProcessConstructor{
-		"Basic": func(ctx context.Context, opts *options.Create) (jasper.Process, error) {
-			mngr, err := jasper.NewSynchronizedManager(false)
-			if err != nil {
-				return nil, errors.WithStack(err)
-			}
+	makeProc := func(ctx context.Context, opts *options.Create) (jasper.Process, error) {
+		mngr, err := jasper.NewSynchronizedManager(false)
+		if err != nil {
+			return nil, errors.WithStack(err)
+		}
 
-			client, err := makeTestServiceAndClient(ctx, mngr)
-			if err != nil {
-				return nil, errors.WithStack(err)
-			}
+		client, err := makeTestServiceAndClient(ctx, mngr)
+		if err != nil {
+			return nil, errors.WithStack(err)
+		}
+
+		return client.CreateProcess(ctx, opts)
+	}
 
-			return client.CreateProcess(ctx, opts)
+	for _, modify := range []struct {
+		Name    string
+		Options testutil.OptsModify
+	}{
+		{
+			Name: "Blocking",
+			Options: func(opts *options.Create) {
+				opts.Implementation = options.ProcessImplementationBlocking
+			},
+		},
+		{
+			Name: "Blocking",
+			Options: func(opts *options.Create) {
+				opts.Implementation = options.ProcessImplementationBasic
+			},
+		},
+		{
+			Name:    "Default",
+			Options: func(opts *options.Create) {},
```

Comment:
This is unnecessary.

Reply:
I know, but making sure that you don't need to set implementation is probably a good thing?

Reply:
This should be left to validation in create options.

Reply:
right but if we're always setting it, then we don't know if the default operation works in this configuration.

Reply:
I guess?